src/supermarket/db/seeds.rb

```diff
@@ -336,4 +338,12 @@
     owner: "chef",
     callback_url: ENV['PUBSUBHUBBUB_CALLBACK_URL'] || 'https://seeded.example.com/curry/pr-updates'
   ).first_or_create!
+
+  unless(QualityMetric.where(name: 'Foodcritic').any?)
+    QualityMetric.create!(name: 'Foodcritic')
+  end
+
+  unless(QualityMetric.where(name: 'Collaborator Number').any?)
+    QualityMetric.create!(name: 'Collaborator Number')
+  end
```

Comment:
🤔  I suspect these "ensure QualityMetrics exists" in seeds should probably be towards the top and outside of the check for development environment. That would follow the philosophy of making seeds responsible for ensuring the creation of The Standard Set of Things that should exist in any running instance of Supermarket.